packages/web/package.json

```diff
@@ -62,17 +62,18 @@
     "promise.allsettled": "1.0.4",
     "prop-types": "15.7.2",
     "raw-loader": "0.5.1",
-    "react": "17.0.2",
+    "react": "^16.14.0",
```

Comment:
We can run into issues with react erroring because of multiple versions of it in the resolution tree. That happened with stems a lot before we synced the versions. If we can continue to use `17.0.2` that would be great!